build-tools/packages/build-cli/src/commands/bundleAnalyses/run.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { execSync } from "child_process";
+import { Flags } from "@oclif/core";
+import { BaseCommand } from "../../base";
+
+export default class BundleAnalysesRun extends BaseCommand<typeof BundleAnalysesRun.flags> {
+    static description = `Run to report the bundle analysis. Do not run Danger directly at the root of the
+    repo as this better isolates its usage and dependencies`;
+
+    static flags = {
+        dirname: Flags.string({
+            description: "Directory",
+            default: __dirname,
```

Comment:
This is possibly a problem long term because this variable is not defined in ESM. I think these solutions will work in both esm and commonjs: https://stackoverflow.com/a/66651120

Reply:
Actually this may be a better API to call: https://nodejs.org/api/url.html#url_url_fileurltopath_url